### ISSUE TITLE
Script fails on accessing Queue

### ISSUE THREAD

**Issue by Enigma0** [Yoshiofthewire/CrunchyXBMC#22]
So everyone has a different queue making this probably a unique problem but in any case:

```
-->Python callback/script returned the following error<--
Error Type: <type 'exceptions.UnicodeEncodeError'>
Error Contents: 'ascii' codec can't encode character u'\u2013' in position 28: ordinal not in range(128)

Traceback (most recent call last):
File "C:\Users\...\AppData\Roaming\Kodi\addons\plugin.video.crunchyroll-takeout\default.py", line 38, in <module>
crunchy_main.main()
File "C:\Users\...\AppData\Roaming\Kodi\addons\plugin.video.crunchyroll-takeout\resources\lib\crunchy_main.py", line 356, in main
check_mode(args)
File "C:\Users\...\AppData\Roaming\Kodi\addons\plugin.video.crunchyroll-takeout\resources\lib\crunchy_main.py", line 331, in check_mode
crj.queue(args)
File "C:\Users\...\AppData\Roaming\Kodi\addons\plugin.video.crunchyroll-takeout\resources\lib\crunchy_json.py", line 908, in queue
% series['name'.encode('utf8')], xbmc.LOGDEBUG)
File "C:\Users\...\AppData\Roaming\Kodi\addons\plugin.video.crunchyroll-takeout\resources\lib\crunchy_json.py", line 1293, in log
xbmc.log(re.sub(rex, "********", msg), level)
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2013' in position 28: ordinal not in range(128)
-->End of Python script error report<--
```

**Comment by Yoshiofthewire** [comment]
Fixed in 2.1.2

**Comment by Enigma0** [comment]
Awesome - will test when able.
